core/key_test.go

```diff
@@ -326,14 +332,14 @@ func TestDecodeIndexDataStoreKey(t *testing.T) {
 }
 
 func TestDecodeIndexDataStoreKey_InvalidKey(t *testing.T) {
-	replace := func(b []byte, i int, v byte) []byte {
+	/*replace := func(b []byte, i int, v byte) []byte {
```

Comment:
todo: Please remove or uncomment the commented out tests.

Reply:
ok